Source/Scene/Scene.js

```diff
@@ -2431,17 +2434,26 @@ define([
                 scene._stencilClearCommand.execute(context, passState);
             }
 
-            if (clearGlobeDepth && useDepthPlane) {
-                depthPlane.execute(context, passState);
-            }
-
-            us.updatePass(Pass.OPAQUE);
-            commands = frustumCommands.commands[Pass.OPAQUE];
-            length = frustumCommands.indices[Pass.OPAQUE];
+            us.updatePass(Pass.CESIUM_3D_TILE_CLASSIFICATION);
+            commands = frustumCommands.commands[Pass.CESIUM_3D_TILE_CLASSIFICATION];
+            length = frustumCommands.indices[Pass.CESIUM_3D_TILE_CLASSIFICATION];
             for (j = 0; j < length; ++j) {
                 executeCommand(commands[j], scene, context, passState);
             }
 
+            // Execute commands in order by pass up to the translucent pass.
+            // Translucent geometry needs special handling (sorting/OIT).
+            var startPass = Pass.CESIUM_3D_TILE_CLASSIFICATION + 1;
+            var endPass = Pass.TRANSLUCENT;
+            for (var pass = startPass; pass < endPass; ++pass) {
+                us.updatePass(pass);
+                commands = frustumCommands.commands[pass];
+                length = frustumCommands.indices[pass];
+                for (j = 0; j < length; ++j) {
+                    executeCommand(commands[j], scene, context, passState);
+                }
+            }
```

Comment:
There is still a lot extra than what's in the original commit: https://github.com/AnalyticalGraphicsInc/cesium/commit/1fda8c0ce8a95e13df150bd686fbfc957823908f

The command execution should not be changed. Maybe it was a bad `git revert`? You can hand edit to fix.